### DISCUSSION TITLE
Option for current context display

### DISCUSSION THREAD

**Discussion by vvvityaaa** [unikubehq/cli#100] [Ideas]
It would be good to have an option under `use`-command to display the context, which is currently set for the user.

**Comment by buschNT** [comment]
Yes, an option to display the current context is missing and would be quite useful.

Maybe we could move all context commands within one group. With the current structure we already have problems with the "--remove" option (see #63)...

**EDITED** based on comments:

- Suggested command structure
```
unikube context set --project --orga --deck
unikube context remove --project --orga --deck
unikube context show
```

- Later on we could add more options here:
```
unikube context list
unikube context save --name
unikube context load --name
unikube context delete --name
```
- show context in command line (edit `PS1`)

Further Context Issues:
- from #63 > Once you delete a project I would expect the context to be unset as well.

What do you think @Schille @SteinRobert @schwobaseggl ?

**Reply by SteinRobert** [reply]
I like the idea. Would add following command to your list:
```bash
unikube context list
```

To display a list of stored contexts.

Another thought on this:

When loading a context we could actually work with bash scripts.
This would allow us to permanently edit `PS1` and achieve something similar like sourcing `bin/activate` from `virtualenv`.

It could look something like this (haven't thought this through obviously):

```bash
robert:projectFolder $ unikube context save --name project.utx  # (utx for "unikube context")
robert:projectFolder $ source project.utx
(projectName) robert:projectFolder $ unikube context show

# beautiful project context information...
```

**Reply by buschNT** [reply]
I updated my post including the list command.

> When loading a context we could actually work with bash scripts.
This would allow us to permanently edit PS1 and achieve something similar like sourcing bin/activate from virtualenv.

We tried that once and it might work. However, we just explored that option and didnt follow up on it at that point in time.

**Reply by Schille** [reply]
Hey @buschNT I stumble upon this issue from time to time. May we move this to an GH issue and work on it?
